sound/soc/sof/pm.c

```diff
@@ -198,29 +198,30 @@ static void sof_suspend_streams(struct snd_sof_dev *sdev)
 {
 	struct snd_sof_pcm *spcm;
 	struct snd_pcm_substream *substream;
-	int dir, playback_dir, capture_dir;
+	int dir, state, playback_dir, capture_dir;
 
-	/* suspend all running streams */
+	/* suspend all running/paused streams */
```

Comment:
@ranj063 if you look at recent patches from Takashi Iwai on alsa-devel, tracked with issue https://github.com/thesofproject/linux/issues/550, the direction is to make snd_pcm_suspend a static function and let the framework deal with PCM suspend. Is this whole sequence really necessary?

Reply:
@plbossart I'll have to check how the framework handles pcm suspend. let me look into it today.

Reply:
@plbossart I've updated the PR now. And we should be good when the change in Takashi's proposal are merged.